### PR TITLE
Presentation Modifications

### PR DESCRIPTION
**Pull request by comsemer**
## Description
* A series of both interns' modifications made during presentation prep phase
* Added some new file-generating services
* Some modifications to control handling
* Removal of certain meta tags
* Modified using statement list generation
* Changed how some directive attributes are handled
* Added some default page route setup pending service usage

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: src/CTA.WebForms2Blazor/Factories/ClassConverterFactory.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ public ClassConverter Build(string sourceFileRelativePath, SemanticModel model,
 
         public IEnumerable<ClassConverter> BuildMany(string sourceFileRelativePath, SemanticModel model)
         {
-            return model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node));
+            return model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node)).ToList();
```
Comment (Eruanion) [inline src/CTA.WebForms2Blazor/Factories/ClassConverterFactory.cs:76]: Null exception potentially here? If the model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node)) resolves to null, if this is possible we should handle a null check here.

File: src/CTA.WebForms2Blazor/Factories/FileConverterFactory.cs [hunk 2]
```diff
@@ -88,7 +94,7 @@ public FileConverter Build(FileInfo document)
 
         public IEnumerable<FileConverter> BuildMany(IEnumerable<FileInfo> documents)
         {
-            return documents.Select(document => Build(document));
+            return documents.Select(document => Build(document)).ToList();
```
Comment (Eruanion) [inline src/CTA.WebForms2Blazor/Factories/FileConverterFactory.cs:97]: Also potential null exception here is documents.select can return null.

**Review by KallanBao — APPROVED** [review]

**Review by Eruanion — CHANGES_REQUESTED** [review]
Good work just a few comments.

File: src/CTA.WebForms2Blazor/Factories/ClassConverterFactory.cs (hunk 1 above)
Comment (comsemer) [reply src/CTA.WebForms2Blazor/Factories/ClassConverterFactory.cs:76]: I'm pretty sure `Select()` (and any Linq methods that return `IEnumerable<T>`) will never return null, if there are no results I'm pretty sure an empty enumerable is returned

**Review by Eruanion — APPROVED** [review]

File: src/CTA.WebForms2Blazor/Services/ProgramCsService.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {{
+                    webBuilder.UseStartup<Startup>();
+                }});
+    }}
+}}";
+
+        public string ProgramCsNamespace { get; set; } = "Unknown_program_cs_namespace";
```
Comment (jonlouie) [inline src/CTA.WebForms2Blazor/Services/ProgramCsService.cs:31]: Same as above, would it be more helpful to change this text to be in command form?

**Review by jonlouie — COMMENTED** [review]
Two minor comments, everything else looks great

**Pull request merged**